templates/message/list.html.twig

```diff
@@ -1,15 +1,19 @@
 {% extends 'message/_layout.html.twig' %}
 
-{% block message_content %}
-    <div class="b__nudge--bottom-small">
-        <form onchange="this.submit();" class="pull-left">
-            <select class="form form__field" name="status">
-                <option value="">Tous les messages</option>
-                <option {{ app.request.query.get('status') == constant('AppBundle\\AdherentMessage\\AdherentMessageStatusEnum::SENT_SUCCESSFULLY') ? 'selected="selected"' }} value="{{ constant('AppBundle\\AdherentMessage\\AdherentMessageStatusEnum::SENT_SUCCESSFULLY') }}">Messages envoyés</option>
-                <option {{ app.request.query.get('status') == constant('AppBundle\\AdherentMessage\\AdherentMessageStatusEnum::DRAFT') ? 'selected="selected"' }} value="{{ constant('AppBundle\\AdherentMessage\\AdherentMessageStatusEnum::DRAFT') }}">Brouillons</option>
-            </select>
+{% block messages_main_block %}
+    <h3>Mes messages <a href="{{ path("app_message_#{message_type}_create", route_params|default({})) }}">+ Nouveau message</a></h3>
+    <div class="background--lighter space--20">
+        <form onchange="this.submit();">
+            <p><strong>Filtrer par...</strong></p>
+            <div><label for="message-status-filter">Statut</label></div>
+            <p>
+                <select class="form form__field" id="message-status-filter" name="status">
+                    <option value="">Tous les messages</option>
+                    <option {{ app.request.query.get('status') == constant('AppBundle\\AdherentMessage\\AdherentMessageStatusEnum::SENT_SUCCESSFULLY') ? 'selected="selected"' }} value="{{ constant('AppBundle\\AdherentMessage\\AdherentMessageStatusEnum::SENT_SUCCESSFULLY') }}">Messages envoyés</option>
```

Comment:
What do you think to pass the `status` variable via the controller instead of to call `app.request.query` here ? 

The variable is already setted : https://github.com/EnMarche/en-marche.fr/pull/3568/files#diff-df4b39e55a63983622cd8af04f040982R49

Reply:
done